webpack/actions/notification.js

```diff
@@ -0,0 +1,26 @@
+import axios from 'axios';
+import routes from '../routes';
+import { READ_NOTIFICATION } from './types';
+
+export function readNotification(notificationIds) {
```

Comment:
This function is written to return a proper flux/redux action, but it isn't invoked that way in NotificationMenu. For now, we might want to refactor this so that it just calls the axios.post.